gateway/src/main/kotlin/GatewayResumeConfiguration.kt

```diff
@@ -0,0 +1,9 @@
+package dev.kord.gateway
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+public data class GatewayResumeConfiguration(
```

Comment:
I think `GatewayConfiguration`, `GatewayResumeConfiguration` and `GatewaySession` should be in the same file.